modules/developer_manual/pages/core/code-standard.adoc

```diff
@@ -0,0 +1,37 @@
+= Code Standards Compliance
+:owncloud-coding-standard-url: https://github.com/owncloud/coding-standard
```

Comment:
missing `:toc: right`
(added a own PR in master for that as that one has already be merged: https://github.com/owncloud/docs/pull/1971)

Reply:
It's not required as the document's too small.